sig/lrama/typeprof-generated.rbs

```diff
@@ -0,0 +1,542 @@
+module Lrama
+  Token: untyped
+  VERSION: String
+  include Comparable
+
+  def to_s: -> String
```

Comment:
I suspect these methods comes from `Lrama::Rule`, which is a struct,
https://github.com/ruby/lrama/blob/4a58968ea5853e7cd53169c15fd0813743ce07da/lib/lrama/grammar.rb#L7. Is this intentional?

Reply:
This is known issue https://github.com/ruby/typeprof/issues/84